portal/src/graphql/portal/LoginIDConfigurationScreen.tsx

```diff
@@ -525,6 +613,19 @@ const AuthenticationLoginIDSettingsContent: React.FC<AuthenticationLoginIDSettin
         <Label className={styles.checkboxLabel}>
           <FormattedMessage id="LoginIDConfigurationScreen.email.domainBlocklist" />
         </Label>
+        <TagPicker
+          inputProps={{
+            "aria-label": renderToString(
+              "LoginIDConfigurationScreen.email.domainBlocklist"
+            ),
+            placeholder: "example.com",
+          }}
+          className={styles.widgetInputField}
+          disabled={!state.email.domain_blocklist_enabled}
+          selectedItems={domainBlocklist}
+          onChange={onDomainBlocklistChange}
+          onResolveSuggestions={onDomainBlocklistSuggestions}
+        />
```

Comment:
TagPicker has a UX problem. If I enter some text and then clear the focus, the text is not "added" to the picker. I think the behavior should be add-item-on-blur. What do you think?

Reply:
I tried use the method we discussed to fix the problem, but still have another problem. When I added the input value to list, I cannot clear the text in input as the input element is controlled by fluent ui. :(

Reply:
Updated, thanks @louischan-oursky !!